servers/physics_2d/godot_area_2d.cpp

```diff
@@ -198,7 +198,7 @@ void GodotArea2D::set_monitorable(bool p_monitorable) {
 }
 
 void GodotArea2D::call_queries() {
-	if (!monitor_callback.is_null() && !monitored_bodies.is_empty()) {
+	if (monitor_callback.is_valid() && !monitored_bodies.is_empty()) {
```

Comment:
This code and the equivalent cases below and in the 3D case is a bit confusing, should probably be cleaned up but didn't do it in this change